uswds/uswds-site
# USWDS-Site: Add clarity to Phase 2: compile instructions

# Summary
Updated the guidance in [Phase 2: Compile](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/) to add clarity by:
1. Re-organizing content to more clearly identify which instructions are for `uswds-compile` users and which are for users who compile Sass a different way
1. Updating the file names in the code demos to match what `uswds-compile` creates
1. Adding copy to clarify which files are project files and which are USWDS files

⚠️ The changelog date will need to be updated before merge.

## Related issue

Closes #2061

## Preview link

Preview link: [Phase 2: Compile](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-update-compile-instructions/documentation/getting-started/developers/phase-two-compile/)

## Problem statement

There is an opportunity to add clarity about which files will be provided by USWDS and which will need to be created by the user. 

This issue was originally flagged in [this Slack thread](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1681831091624819).

## Testing and review
- Confirm that there are no spelling or grammatical errors
- Confirm the file names in the code samples matches what is generated by `uswds-compile` 
- Confirm that the content order makes sense
